nix_update/__init__.py

```diff
@@ -32,6 +32,11 @@ def parse_args() -> Options:
         help="Regex to extract version with, i.e. 'jq-(.*)'",
         default="(.*)",
     )
+    parser.add_argument(
+        "--stable-version",
+        action="store_true",
+        help="Only use semantic versions without build or pre-release suffixes",
```

Comment:
I wonder if the behavior should be inverted and the flag could be called `unstable` or `prerelease`.

Reply:
Oh, we could do that and later expand or introduce a related flag to fetch the HEAD commit from a branch!

Reply:
Right.